### PR TITLE
feat: added stacked cards

### PR DESCRIPTION
**Pull request by suyogkokaje**

### CODE REVIEW

**Comment by suyogkokaje** [comment]
@thepranaygupta please have a look.

File: Stacked Cards/style.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@200;300;400;500;600;700;800;900&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
+
+body{
```
Comment (Ovenoboyo) [inline Stacked Cards/style.css:10]: Use css classes instead of directly styling html tags

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

**Comment by suyogkokaje** [comment]
@Ovenoboyo please have a look

File: Stacked Cards/style.css [hunk 2]
```diff
@@ -4,19 +4,23 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-family: 'Poppins', sans-serif;
 }
 
 body{
+    background: #2e557c;
+}
```
Comment (thepranaygupta) [inline Stacked Cards/style.css]: add this `background` style to the `parent` class not to the body tag

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

**Comment by suyogkokaje** [comment]
@thepranaygupta please have a look

**Review by Abirpal202049 — COMMENTED** [review]
Add a dummy image in the card and give some line-height after the Card Heading and make the text-align: justify; for the body of the card

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]
As @Abirpal202049 suggested

Add a dummy image in the card and give some line-height after the Card Heading and make the text-align: justify; for the body of the card

**Comment by suyogkokaje** [comment]
@Abirpal202049 please have a look

**Review by Ovenoboyo — APPROVED** [review]

**Review by Abirpal202049 — APPROVED** [review]

**Comment by Abirpal202049** [comment]
Add the Readme.md of your Project

**Comment by suyogkokaje** [comment]
@Abirpal202049 I have added readme file

**Review by Abirpal202049 — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]
LGTM

**Review by Ovenoboyo — APPROVED** [review]

**Review by saikatmu31 — APPROVED** [review]

**Pull request merged**